keytar/index.d.ts

```diff
@@ -56,4 +56,4 @@ export declare function replacePassword(service: string, account: string, passwo
  *
  * @returns the string password or null on failures.
```

Comment:
Not sure I understand - this method seems correct: https://github.com/atom/node-keytar/blob/master/src/main.cc#L38